src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/RuleBasedTests.cs

```diff
@@ -232,10 +254,14 @@ public async Task CheckOnePatient(string templateName, string samplePath, DataTy
         {
             var result = await ConvertData(templateName, samplePath, dataType);
             var patients = result.SelectTokens("$.entry[?(@.resource.resourceType == 'Patient')].resource.id");
-            if (templateName == "ADT_A40" || templateName == "ADT_A41")
+            if (templateName == "ADT_A40" || templateName == "ADT_A41" || templateName == "ADT_A45" || templateName == "ADT_A47")
```

Comment:
If the special templates here will increase all the time, can we consider using list to store them?

Reply:
ok sure, will store templates which are having no patient resource in a list. I will update it and will push the changes

Reply:
@yankunhuang-pku : We have updated the RuleBasedTests file to contain the templates name in list for both non and multiple patient resource